app/notifications/user_sing_up_notification.rb

```diff
@@ -0,0 +1,35 @@
+class UserSingUpNotification < Noticed::Base
```

Comment:
```suggestion
class UserSignUpNotification < Noticed::Base
```

Spelling error in class